PROJECT

```diff
@@ -13,6 +13,10 @@ resources:
   kind: EnvoyFleet
   path: github.com/kubeshop/kusk-gateway/api/v1alpha1
   version: v1alpha1
+  webhooks:
+    defaulting: true
```

Comment:
defaulting is for mutating webhook, isn't it?

Reply:
Fixed